src/main/java/hudson/plugins/s3/Entry.java

```diff
@@ -41,6 +41,11 @@
     public boolean noUploadOnFailure;
 
     /**
+     * Fail the build if no files were uploaded.
+     */
+    public boolean failNoFilesToUpload;
```

Comment:
I think I messed up the merge commit. It looks like that reverted the code change in the `config.jelly` file.